src/actions/render.js

```diff
@@ -43,6 +44,7 @@ const renderLocalizationView = (view, outputType, language, basePath) => {
   return localizationTemplate(outputType)
     .map((source) => Handlebars.compile(source))
     .map((template) => template(view))
+    .map((renderedTemplate) => postTemplateRenderFormatting(renderedTemplate, outputType))
```

Comment:
Instead of `postTemplateRenderFormatting` can we rename it to something indicating what this function does? something like `unflattenJson` or something.

Reply:
Then it's very specific. In this function, we could be doing different things based on output mode,

Reply:
> we could be doing different things based on output mode
Then probably it's against "single responsibility" principle :D

Reply:
Ain't it single resposibility, If the responsibility is "postProcessing" whatever that means based on outputType? o_O

Reply:
I agree with @pooyahrtn, let's be explicit about what the function is doing. If we ever need more complicated post-processing steps, we can always change the function name later on.